src/main/resources/application.properties.example

```diff
@@ -0,0 +1,12 @@
+server.port=8081
+# spring datasource url, username, pass
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/tutor_media?createDatabaseIfNotExist=true&useUnicode=yes&characterEncoding=UTF-8&useSSL=false
+spring.datasource.username={username}
+spring.datasource.password={password}
+
+# spring hibernate dialect for Mysql
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
+
+# spring jpa ddl-auto for validating db-schema
+spring.jpa.hibernate.ddl-auto=validate
```

Comment:
why validate?? we can use create here since its in development phase.. When its in production then we can use none here. Its recommended to use create when its in test phase because it might be possible that we update the model.

Reply:
@subratanath123 
For controlling over db schema (create & update & delete) we already hand-over this to flyway. I use validate for just validate schema in development. In production, it should be an empty string that's logical. but create-drop i used in testing environment where all mock data already ready for fire in every startup. Though i am not sure what will be best practice for dev or production or testing.